spec/dummy/app/models/sibling.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: people
+#
+#  id         :bigint           not null, primary key
+#  name       :string
+#  type       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  user_id    :bigint
+#  person_id  :bigint
+#
+class Sibling < Person
```

Comment:
Doesn't `Spouse` do the same thing?

Reply:
Sure it does , but I wanted to make more clearer that you can address more than one resource in the same has_many field